addons/project/views/project_views.xml

```diff
@@ -495,12 +520,17 @@
                     <field name="analytic_account_id"/>
                     <field name="stage_id" groups="project.group_project_stages"/>
                     <field name="tag_ids"/>
+                    <filter string="My Projects" name="own_projects" domain="[('user_id', '=', uid)]"/>
                     <filter string="My Favorites" name="my_projects" domain="[('favorite_user_ids', 'in', uid)]"/>
-                    <separator/>
                     <filter string="Followed" name="followed_by_me" domain="[('message_is_follower', '=', True)]"/>
-                    <filter string="My Projects" name="own_projects" domain="[('user_id', '=', uid)]"/>
+                    <filter string="Unassigned" name="unassigned_projects" domain="[('user_id', '=', False)]"/>
+                    <separator/>
+                    <filter string="Late Milestones" name="late_milestones" domain="[('is_milestone_exceeded', '=', True)]"/>
+                    <separator/>
+                    <filter string="Rated Projects" name="rated_projects" domain="[('task_ids.rating_last_value', '!=', 0.0)]" groups="project.group_project_rating"/>
```

Comment:
This filter will be costly on big databases. @aju-odoo agreed to remove it.
Thanks!